Controller/SnippetAreaController.php

```diff
@@ -113,5 +128,15 @@ public function getAction(Request $request, string $area): Response
                 'Content-Type' => 'application/json',
             ]
         );
+
+        // cache-control directives
+        $response->setPublic();
+        $response->setMaxAge($this->maxAge);
+        $response->setSharedMaxAge($this->sharedMaxAge);
+
+        // set reverse-proxy TTL (Symfony HttpCache, Varnish, ...) to avoid caching of intermediate proxies
+        $response->headers->set(SuluHttpCache::HEADER_REVERSE_PROXY_TTL, $this->maxAge);
```

Comment:
code overall looks good - but i am not sure if we should have another configurable cachelifetime for the reverse proxy cache. @alexander-schranz what do you think?

Reply:
I agree, I was not sure how to handle the reverse proxy cache lifetime ...
Another point I'm not sure is: is it possible for some people do disable the sulu httpcache bundle ? In which case maxAge and sharedMaxAge won't be defined.

Reply:
@alexander-schranz ping :)